js/components/UserList.js

```diff
@@ -0,0 +1,82 @@
+import { getUserList, getUser, addUser } from '../api/todoAPI.js';
+
+export default function UserList($element, {setUserInfo} ) {
+    this.users = [];
+    this.activeUser = {}
+    this.initialIdForTest = 'woSRmp5oh';
+
+    this.getUserList = async () => {
+        try{
+            this.users = await getUserList();
+        }catch(err){
+            console.log("err", err);
```

Comment:
```suggestion
            console.error("err", err);
```
에러임을 좀 더 명확하게 표시하기 위해 단순 로그보단 `error`를 사용해보는 건 어떨까요?

Reply:
평소 로그에 에러를 사용해보지 않아서 생각을 못했었는데 다음부터 한번 사용해보도록 하겠습니다 :)

Reply:
오 평소 잘 쓰지 않아 생각을 못했네요 ㅜ. 언급해주셔서 감사합니다.